sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Assessments"
+        :routeLocation="goBackRouteParams"
+        subTitle="View Request"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestDetails.status ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithSABC
+          "
+        >
+          <v-row class="p-0 m-0">
+            <v-btn color="primary" variant="outlined"
+              >Decline reassessment</v-btn
+            >
+            <v-btn class="ml-2" color="primary">Approve reassessment</v-btn>
+          </v-row>
+        </template>
+      </header-navigator>
+      <application-offering-change-details-header
+        class="m-4"
+        :headerDetails="headerDetails"
+      />
+    </template>
+    <template
+      #alerts
+      v-if="
+        applicationOfferingChangeRequestDetails.status ===
+        ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+      "
+      ><application-offering-change-request-banner />
+    </template>
+    <template #tab-header>
```

Comment:
This is not actually a header, right? It is part of the page content, do we need a slot for it?

Reply:
I discussed this with @andrewsignori-aot . The conclusion was not to modify the full page container for this addition as it is the only addition.

Reply:
I need to recall the discussion, but I believe that I mentioned that adding a new slot to the full page container would not be good but I do not recall saying that leaving `student-application-offering-change-details` inside `template #tab-header` would be ok.
Is this just not part of the `full-page-container` "body"?